e2e/config/e2e_load_test.config.tmpl.yaml

```diff
@@ -28,28 +28,48 @@ e2e_load_test_config:
   # Load Test Mode
   # Tests run against a specified gateway URL (local or public)
   # Examples:
-  #   - Local:  "http://localhost:3091/v1"
   #   - Public: "https://rpc.grove.city/v1"
+  #   - Local:  "http://localhost:3091/v1"
   load_test_config:
     # [Required] Custom PATH gateway URL
-    gateway_url_override: ""
+    #
+    # The default field uses the Grove Portal as the gateway URL.
+    #
+    # If you are wishing to test against a local PATH instance,
+    # you can override this field with the local PATH gateway URL 
+    # (default: "http://localhost:3069/v1")
+    gateway_url_override: "https://rpc.grove.city/v1"
+
+    # [Optional] Insert service ID into subdomain per test case
+    #     Example: "F00C.rpc.grove.city/v1"
+    #
+    # TODO_TECHDEBT(@commoddity): Remove after header-based service selection is supported
```

Comment:
Can you #PUC on the TODO here? Why? What is it?

Reply:
It links to the issue that was created to capture this context so IMO we can just keep the comment to direct the user there if they need to learn more.